process/block/shardblock.go

```diff
@@ -936,6 +937,27 @@ func (sp *shardProcessor) displayPoolsInfo() {
 		"total miniblocks", sp.dataPool.MiniBlocks().Len(),
 		"miniblocks pool capacity", sp.dataPool.MiniBlocks().MaxSize(),
 	)
+
+	for _, hash := range sp.dataPool.MiniBlocks().Keys() {
```

Comment:
this is duplicated code from metachain. put it to base.

Reply:
Done